### PR TITLE
RMT production with reference CI 4476

### PR DESCRIPTION
**Pull request by sandrew**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
@fschueller will review your pull request :dancers: check https://trello.com/c/mQ52gd2m/4476-5-rmt-reference-host
  @fschueller: Please review this pull request using our guidelines:
  * test for acceptance criteria / functionality
  * check if the new code is covered with tests
  * check for unintended consequences
  * encourage usage of the boyscout rule
  * make sure the code is architected in the best way
  * check that no unnecessary technical debt got introduced
  * make sure that no unnecessary FIXMEs or TODOs got added

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-* System dependencies
+## Configuration
 
-* Configuration
+Available configuration options can be found in `config/rmt.yml` file.
 
-* Database creation
+- `mirroring.base_dir` - a directory where repos' files will be stored. Should be under HTTP server public root directory.
+- `mirroring.mirror_url_prefix` - subpath to access repos's files for HTTP server.
+- `mirroring.mirror_src` - whether to mirror `scr` and `debug` repos or not.
```
Comment (ikapelyukhin) [inline README.md]: Just `src`. DebugInfo repos are chosen via CLI.

File: config/rmt.yml [hunk 2]
```diff
@@ -12,5 +12,5 @@ http_client:
   proxy_password:
 
 scc:
-  username:
-  password:
+  username: <%= ENV['RMT_ORGANIZATION_USERNAME'] %>
+  password: <%= ENV['RMT_ORGANIZATION_PASSWORD'] %>
```
Comment (ikapelyukhin) [inline config/rmt.yml]: Hmm, I'm not sure if we want that for the release RPM package.

File: bin/rmt-mirror [hunk 3]
```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env ruby
 
-require_relative '../config/boot'
-$LOAD_PATH.unshift File.expand_path(File.join(__dir__, '../lib/'))
+require_relative '../config/environment'
```
Comment (ikapelyukhin) [inline bin/rmt-mirror]: I like it, but `time bin/rmt-repos` went from 0.8 sec to 1.5 sec (which doesn't do anything and just prints help message), which is somewhat annoying. :-/
We don't need to load all of Rails dependencies, just config and ActiveRecord.

File: config/rmt.yml (hunk 2 above)
Comment (sandrew) [reply config/rmt.yml]: It's not usable without editing anyway, so I don't see any sense at keeping it old way either

Comment (ikapelyukhin) [reply config/rmt.yml]: OK! :+1:

**Pull request merged**